.jscsrc

```diff
@@ -2,5 +2,8 @@
   "preset": "google",
   "validateLineBreaks": "LF",
   "validateIndentation": 2,
-  "excludeFiles": ["node_modules/**"]
+  "excludeFiles": ["node_modules/**"],
+  "disallowSpacesInAnonymousFunctionExpression": {
+    "beforeOpeningRoundBrace": null
+  }
```

Comment:
No, for some odd reason, that doesn't work. The change in the pull request does work, which is weird because my assumption would have been that they're pretty much the same but I guess not.